src/templates/event-index.tsx

```diff
@@ -69,25 +69,28 @@ const EventIndex = ({ data }: EventIndexProps) => {
               )
             }
 
-            let speakers_list = null
-            let speakers: JSX.Element | undefined
-            const speakers_data = item.speakers ? item.speakers : []
-            if (item.speakers) {
-              speakers_list = speakers_data.map((p, i) => {
-                return (
-                  <Person
-                    key={`p${i}`}
-                    person={p?.data as unknown as PersonComponentProps}
-                    type="index"
-                  />
-                )
-              })
-              speakers = (
-                <div className="speakers hidden">
-                  <ul className="inline-list">{speakers_list}</ul>
-                </div>
-              )
-            }
+            // TODO: The speakers list is not currently being displayed. Uncomment this code to display the list and also add to return statement below
```

Comment:
One additional thing that Typescript checking turned up was a few places where we are going to the trouble of generating a chunk of a page but not returning/rendering it (probably because of data quality issues. I tried to mark these with some TODOs but I probably didn't do them all